web/components/home/index.sass

```diff
@@ -0,0 +1,21 @@
+.homepage
+  background: $gradient-purple
+
+  &-email-registration
+    @media screen and (max-width: 300px)
```

Comment:
Can we follow google chrome standard and use `$mobile-s` for `320px`?

Reply:
the 300px is from figma - should I change it anyway?

Reply:
yeah, small phones like "iPhone SE" have a width of 320px